sdk/src/manifest.rs

```diff
@@ -295,10 +312,14 @@ impl Manifest {
                     let ingredient = Ingredient::from_ingredient_uri(store, &assertion_uri)?;
                     manifest.add_ingredient(ingredient);
                 }
-                Actions::LABEL => {
-                    let actions = Actions::from_assertion(assertion)?;
-                    manifest.add_assertion(&actions)?; // assertion.as_json_object()?)?;
-                }
+                // Actions::LABEL => {
+                //     let actions = Actions::from_assertion(assertion)?;
+                //     let ma = ManifestAssertion::new(label, value)
+                //                 .set_instance(claim_assertion.instance())
+                //                 .set_kind(ManifestAssertionKind::Cbor);
+
+                //     manifest.add_assertion(&actions)?.set_instance(claim_assertion.instance()); // assertion.as_json_object()?)?;
+                // }
```

Comment:
Did you mean to take this out? I feel like it is cleaner to take things out and reference VCS instead of leaving commented-out blocks in the codebase if at all possible.

Reply:
yes I'll take it out. But Ideally we would have validation for all the supported types.